docs/database/elasticsearch.md

```diff
@@ -32,7 +32,7 @@ If the endpoint id is not available, the span name SHOULD be the `http.request.m
 | [`db.operation`](database-spans.md) | string | The endpoint identifier for the request. [4] | `search`; `ml.close_job`; `cat.aliases` | Required |
 | [`db.statement`](database-spans.md) | string | The request body for a [search-type query](https://www.elastic.co/guide/en/elasticsearch/reference/current/search.html), as a json string. | `"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"` | Recommended: [5] |
 | `http.request.method` | string | HTTP request method. [6] | `GET`; `POST`; `HEAD` | Required |
-| [`server.address`](../general/attributes.md) | string | Server address - domain name if available without reverse DNS lookup, otherwise IP address or Unix domain socket name. [7] | `example.com` | See below |
+| [`server.address`](../general/attributes.md) | string | Server address - domain name if available without reverse DNS lookup, otherwise IP address or Unix domain socket name. [7] | `example.com` | Recommended |
```

Comment:
the "See below" was inherited from the constraint on general db semconv, but no note was added on elastic semconv.

/cc @estolfo

Reply:
created https://github.com/open-telemetry/semantic-conventions/issues/360 to track it